server/store.go

```diff
@@ -123,6 +135,7 @@ func (s *storeInfo) getLocationID(keys []string) string {
 type StoreStatus struct {
 	*pdpb.StoreStats
 
+	Blocked           bool      `json:"blocked"`
```

Comment:
Yep, we can check the scheduler's name to detect conflict too, but the suggested function name seems weird to me.